streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -75,47 +76,69 @@
 
     private static final long STREAMS_CONSUMER_TIMEOUT = 2000L;
     private static final long CLEANUP_CONSUMER_TIMEOUT = 2000L;
+    private static final int TIMEOUT_MULTIPLYER = 5;
 
     private final WaitUntilConsumerGroupGotClosed consumerGroupInactive = new WaitUntilConsumerGroupGotClosed();
-
-    private AdminClient adminClient = null;
+    private static int testNo = 0;
```

Comment:
Not sure. But I did write the test... Anyway. Main reason for this is, to avoid code duplication and to reuse as much  as possible of the test setup code.